compositeviews/dataset.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@id": "connectome-test-search-composite-01",
+  "@type": "CompositeView",
+  "sources": [
+    {
+      "@id": "test-source-01",
+      "@type": "ProjectEventStream"
+    }
+  ],
+  "projections": [
+    {
+      "@id": "connectome-projection-composite-01",
+      "@type": "ElasticSearchProjection",
+      "mapping": {
+          "dynamic": false,
+          "properties": {
+            "@type": {
+                "type": "keyword"
+            },
+            "name": {
+                 "type": "text"
+            },
+            "inLanguage": {
+                "type": "text"
+            },
+            "abstract_en": {
+              "type": "text"
+            }
+        }
+      },
+      "query": "PREFIX schema: <http://schema.org/> PREFIX nxv: <https://bluebrain.github.io/nexus/vocabulary/> CONSTRUCT {  {resource_id} ?p ?o .  ?author ?pAuthor ?oAuthor .  ?place ?pPlace ?oPlace .} WHERE { {resource_id} ?p ?o .  FILTER(!strstarts(str(?p),str(nxv:)))  OPTIONAL {  {resource_id} schema:author ?author .  ?author ?pAuthor ?oAuthor .  FILTER(!strstarts(str(?pAuthor),str(nxv:)) && ?oAuthor != 'validate' && ?oAuthor != 'validate@validate.com')}    OPTIONAL {    {resource_id} schema:spatialCoverage ?place .        ?place ?pPlace ?oPlace .        FILTER(!strstarts(str(?pPlace),str(nxv:)) && ?oPlace != 'validate')    }}",
```

Comment:
Wie besprochen wäre es schön die Query im eigenen File zu haben zwecks einfacher Les- und Wartbarkeit.

Reply:
I created an issue for this: https://github.com/Connectome-Consortium/rescs_shacl_shapes/issues/16